packages/interbit-ui-tools/src/middleware/sagas/chains.js

```diff
@@ -208,25 +217,21 @@ function* loadChain({ cli, chainAlias, chainId }) {
 
   yield put.resolve(actionCreators.chainLoaded({ chainAlias, chainId }))
 
-  return chain
+  yield call(detectBlocking, chain)
+
+  yield put.resolve(actionCreators.chainBlocking({ chainAlias }))
```

Comment:
Wait for chain to start blocking, then dispatch the chainBlocking action. detectBlocking will throw if the chain does not start blocking.